examples/turn-server/simple/Dockerfile

```diff
@@ -1,7 +1,28 @@
 # SPDX-FileCopyrightText: 2023 The Pion community <https://pion.ly>
 # SPDX-License-Identifier: MIT
 
-FROM golang:latest
+##### builder
```

Comment:
I think this comment is superfluous. The `AS builder` should be sufficient

Reply:
```suggestion
```

Reply:
Fixed in [3f83860](https://github.com/pion/turn/pull/357/commits/3f83860de7001522e614951e959766dba7db37bf)